builtin/logical/pki/path_fetch_issuers.go

```diff
@@ -0,0 +1,258 @@
+package pki
+
+import (
+	"context"
+	"encoding/pem"
+	"regexp"
+	"strings"
+
+	"github.com/hashicorp/vault/sdk/framework"
+	"github.com/hashicorp/vault/sdk/logical"
+)
+
+var nameMatcher = regexp.MustCompile("^" + framework.GenericNameRegex("ref") + "$")
+
+func pathListIssuers(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "issuers/?$",
+
+		Callbacks: map[logical.Operation]framework.OperationFunc{
```

Comment:
I believe this is the deprecated way to specify framework.Paths? Could we update to adopt the new way, such as https://github.com/hashicorp/vault/blob/main/builtin/logical/pki/path_root.go#L28-L43

Reply:
Should we do this afterwards? On `main` I see:

```
[cipherboy@xps15 pki]$ gif 'Operations: ' | wc -l
8
[cipherboy@xps15 pki]$ gif 'Callbacks: ' | wc -l
17
```

which to me says we'd benefit from an all-at-once move?